e2e/webpack_devserver/pnpm-workspace.yaml

```diff
@@ -1,2 +1,3 @@
 packages:
     - '.'
+    - 'lib/**'
```

Comment:
Can we avoid the double-star and just use `*`?

Reply:
✅